frontend/src/locales/en.json

```diff
@@ -201,13 +201,14 @@
       "uppercase": "One uppercase letter required."
     },
     "thx": {
-      "activateEmail": "Your account has not been activated yet, please check your emails and click the activation link!",
+      "activateEmail": "Your account has not been activated yet, please check your emails and click the activation link! Or order a new activation link over the password reset page.",
       "checkEmail": "Your email has been successfully verified. You can sign in now.",
       "email": "We have sent you an email.",
       "emailActivated": "Thank you your email has been activated.",
       "errorTitle": "Attention!",
       "register": "You are registered now, please check your emails and click the activation link.",
       "reset": "Your password has been changed.",
+      "resetPassword": "The code you used was to old please order a new on over the password reset page.",
```

Comment:
where is this used? Can we have a button for that case aswell or is that already done?

Reply:
It is not used anymore since we send the user to forgot-password as soon as the OptIn code is not valid anymore!
So I will delete this and in the /thx page the resetFailed block where this was used.

Reply:
P.S: The button is defined in the thx page as config block:
```
  resetFailed: {
    headline: 'site.thx.errorTitle',
    subtitle: 'site.thx.resetPassword',
    button: 'settings.password.reset',
    linkTo: '/forgot-password',
  },
```